### DISCUSSION TITLE
Lab: Data Management Basics

### DISCUSSION THREAD

**Discussion by mbcann01** [brad-cannell/intro.r.public#32] [Labs]
Please feel free to post questions about this lab below. You are also welcome to leave comments below about the usefulness of this lab. Which parts were helpful? What could I do to improve it? What is still unclear?

**Comment by rsanders3** [comment]
Hello! I am not sure if it was an error on my end, but I cannot seem to access Chapter 25: Introduction to Data Management. I receive an error message saying the page is not found when I try to click on the link for the chapter. Please see my screenshot for the exact message that appears.
![Screenshot 2025-06-23 at 11 51 34 AM](https://github.com/user-attachments/assets/29570472-1a34-463d-8609-eb973adbea0f)

**Reply by mbcann01** [reply]
Hi @rsanders3 ,

Thank you for letting me know! 

I haven't quite figured out why this is happening, but I'm working on it.

**Reply by mbcann01** [reply]
Hi @rsanders3 ,

I think this should be fixed now. I apologize for the inconvenience. Please let me know if you run into any other issues.

**Reply by rsanders3** [reply]
Yes, it is working! Thank you so much!
